### PR TITLE
MIMIR-137 move clear cache over to statistics tab

### PR DESCRIPTION
**Pull request by Garlov**
MIMIR-137

### CODE REVIEW

File: src/main/resources/react4xp/_entries/Dashboard/containers/DashboardTools/index.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    return (<Trash size={16}/>)
+  }
+
+  return (
+    <div className="p-4 tables-wrapper">
+      <h2>Diverse verktøy</h2>
+      <Container>
+        <Row>
+          <Col>
+            <span>Tøm Cache</span>
```
Comment (johnnadeluy) [inline src/main/resources/react4xp/_entries/Dashboard/containers/DashboardTools/index.jsx:32]: Is the text supposed to be outside of the button? (See image attachment in JIRA case)

**Review by johnnadeluy — CHANGES_REQUESTED** [review]
There is a difference between the image attachment and the result. I'll further comment on this in the JIRA case.

File: src/main/resources/react4xp/_entries/Dashboard/containers/DashboardTools/index.jsx (hunk 1 above)
Comment (Garlov) [reply src/main/resources/react4xp/_entries/Dashboard/containers/DashboardTools/index.jsx:32]: Yes

**Review by johnnadeluy — APPROVED** [review]
Good job! 🙌

**Pull request merged**